reference/features/filtering_and_faceted_search.md

```diff
@@ -0,0 +1,318 @@
+# Filtering and faceted search
+
+You can use MeiliSearch's filters to refine search results and create faceted search interfaces.
+
+## Configuring filters
+
+Filters use [document fields](/learn/core_concepts/documents.md#fields) to establish filtering criteria.
+
+To use a document field as a filter, you must first add its attribute to the [`filterableAttributes` index setting](/reference/api/filterable_attributes.md).
+
+**This step is mandatory and cannot be done at search time.** Filters need to be properly processed and prepared by MeiliSearch before they can be used, an operation that takes as much time as indexing all your documents.
+
+::: warning
+By default, `filterableAttributes` is empty. This means that filters do not work without first explicitly adding attributes to the `filterableAttributes` list.
+:::
+
+Filters work with numeric and string values. Empty fields or fields containing an empty array will be ignored. Fields containing object values will throw an error.
+
+### Example
+
+Suppose you have a collection of movies containing the following fields:
+
+```json
+[
+  {
+      "id": "458723",
+      "title": "Us",
+      "director": "Jordan Peele",
+      "genres": [
+        "Thriller",
+        "Horror",
+        "Mystery"
+      ],
+      "overview": "Husband and wife Gabe and Adelaide Wilson take their […]",
+  },
+  …
+]
+```
+
+If you want to filter results based on the `director` and `genre` attributes, you must add them to the [`filterableAttributes` list](/reference/api/filterable_attributes.md).
+
+<CodeSamples id="faceted_search_update_settings_1" />
+
+## Using filters
+
+Once you have configured `filterableAttributes`, you can start using [the `filter` search parameter](/reference/features/search_parameters#filter). Search parameters are added to a query when a user searches for a document.
+
+`filter` expects a **filter expression** containing one or more **conditions**. A filter expression can be written as a string, as an array, or as a mix of both.
+
+::: warning
+`filter` is a new search parameter introduced in MeiliSearch v0.21. It combines functionality previously present in v0.20's `filters` and `facetFilters`.
+
+Both `filters` and `facetFilters` have been removed from v0.21.
+:::
```

Comment:
Since we are not in v1 we do not provide support for the previous MeilISearch versions. To avoid additional maintaining I think it should be removed.
```suggestion
```